### DISCUSSION TITLE
Offload generation process to another device

### DISCUSSION THREAD

**Discussion by LostRune** [stashapp/stash#3926] [Q&A]
Is it possible to offload the generation process to another device?
Or basically to generate everything on one device and copy everything to another without conflicts?

Mostly because my stash-device is not that powerful.
I tried setting up another stash instance on the other device, but then all scene-ids are different.

I appreciate any help/suggestions.

**Comment by sleetx** [comment]
I suppose you could back up the Stash database on one device and restore it on another... But why not just host Stash from the device capable of the generation? You can access the Stash content through a web browser on your other device(s).

**Reply by LostRune** [reply]
Already tried the backup/restore method, but then all the paths are incorrect, because the media is not on the same device anymore but rather only reachable over the network via a different path (stash and media files are on a NAS device). 

The problem is, that I want to host stash on the 24/7 online NAS and generate the files on the more powerful PC.

Thanks for the suggestions anyway!

**Comment by SH0D4N-1337** [comment]
Hi @LostRune, have you found a way to do this? Only solution I thought about is using Docker.

Example:
Docker on PC: `\\SMB-to-NAS\Stash:/data`
Docker on NAS `/NAS/Stash:/data`

After moving to NAS it should look all the same to the Docker.

**Reply by LostRune** [reply]
Your solution should work, but I would advice to shutdown the stash instance on the NAS while generating it on the PC to avoid concurrency issues as stated above.

I personally decided this is not worth the time investment and didn't look more into this issue.
Sorry.
